openfisca_uk/parameters/benefits/universal_credit/means_test/earn_disregard_with_housing.yaml

```diff
@@ -0,0 +1,7 @@
+description: Earnings disregard for means testing Universal Credit per month
```

Comment:
Are earnings disregards equivalent to personal allowances (e.g. as the housing benefit refers to)?